src/pages/Actions/utils.ts

```diff
@@ -35,13 +36,23 @@ const getRequiredFee = (queueLength: BigNumber): BigNumber => {
 export const getCompoundingQueueLength = async (
   web3: Web3
 ): Promise<BigNumber> => {
-  const queueLengthHex = await web3.eth.getStorageAt(
-    COMPOUNDING_CONTRACT_ADDRESS!,
-    0 // EXCESS_WITHDRAWAL_REQUESTS_STORAGE_SLOT
-  );
-
-  if (!queueLengthHex)
-    throw new Error('Unable to get compounding queue length');
+  let queueLengthHex;
+  try {
+    queueLengthHex = await web3.eth.getStorageAt(
+      COMPOUNDING_CONTRACT_ADDRESS!,
+      0 // EXCESS_WITHDRAWAL_REQUESTS_STORAGE_SLOT
```

Comment:
Minor stuff, but this should be `EXCESS_CONSOLIDATION_REQUESTS_STORAGE_SLOT` instead here.

Reply:
Good spot, will note this and patch separate before mainnet